packages/button/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.2.0",
   "main": "dist/main.js",
   "license": "MIT",
+  "typings": "definitions/index.d.ts",
```

Comment:
as you know there is build process from very beggining but its building test files instead of production bundles to dist folder, not sure how this is related to #153 if it is related somehow  then you should wait before #153 is closed and even then add it to the template as well